pilot/pkg/security/authn/utils/utils_test.go

```diff
@@ -32,12 +33,20 @@ import (
 	"istio.io/istio/pkg/spiffe"
 )
 
+const (
```

Comment:
This list is a subset of the default above.  Explain why?

It doesn't seem like the tests try each of these ciphers individually to validate that they work. Tests also don't seem to perform a negative test to ensure mismatches fail - which they should.

Reply:
The list in tlsInboundCiphers1 is just a few ciphers that are used in unit tests to test the Envoy config generated is as expected; the list in tlsInboundCiphers1 can be any ciphers allowed by Envoy. I will add a comment to explain it. 
The negative tests and tests validate the ciphers work will be in separate PRs.